docs/maproulette_upload.md

```diff
@@ -0,0 +1,49 @@
+# MapRoulette Upload Command
+
+The [MapRouletteUploadCommand](/src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java) deserilizes line-delimited Atlas Check flags, and uploads them to the provided MapRoulette server.
+
+The command creates a new Challenge for each Atlas Check and country in the input data. For example: the following Challenges are created if the input data contains flags for EdgeCrossingEdgeCheck and SinkIslandCheck in USA and CAN:
+
+1. CAN - EdgeCrossingEdgeCheck
+2. CAN - SinkIslandCheck
+2. USA - EdgeCrossingEdgeCheck
+3. USA - SinkIslandCheck
+
+Each Challenge is added to the project provided in the MapRoulette connection url.
+
+## Parameters
+
+* **maproulette** - MapRoulette connection url
+* **logfiles** - Path to `/flag` output geojson
+* **config** - Path to Atlas Checks configuration
+* **countries** (optional) - Comma delimited list of country permit list
+* **checks** (optional) - Comma delimted list of checks permit list
+* **outputPath** (optional) - Path to store text file of new challenge ids
+* **checkinComment** (optional) - Value of Changeset check in comment
+* **checkinCommentPrefix** - Prefix added to default check in comment of [prefix ISO - CheckName]
+
+## Example
+
+The following command will upload EdgeCrossingEdge & SinkIsland checks to the `checks_example_project` Project on staging.maproulette.org. 
+Please remember to change the MapRoulette API key. You can find this in your MapRoulette user settings.
+
+```java
+java -cp build/libs/atlas-checks-6.1.3-SNAPSHOT-shaded.jar org.openstreetmap.atlas.checks.maproulette.MapRouletteUploadCommand \
+-maproulette="https://staging.maproulette.org:443:checks_example_project:api|key" \
```

Comment:
I don't think you want "staging" in there. I had to remove it to make it work for me.
```suggestion
-maproulette="https://maproulette.org:443:checks_example_project:api|key" \
```